multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "find_shortest_paths.py", line 284, in find_paths_of_interest
    w1=1+1/round(aggregate_edge_data(path_d_data1,'weight'),3)
ZeroDivisionError: division by zero
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "find_shortest_paths.py", line 432, in <module>
    find_shortest_paths(args.rdfpath,args.modelpath,args.graphpath,graph_types[args.graphtype],args.embedpath,"tfcg",args.fcgtype,args.fcgclass,args.cpu)
  File "find_shortest_paths.py", line 404, in find_shortest_paths
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "find_shortest_paths.py", line 404, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
ZeroDivisionError: division by zero

real	1m15.749s
user	51m44.219s
sys	0m42.511s
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "find_shortest_paths.py", line 284, in find_paths_of_interest
    w1=1+1/round(aggregate_edge_data(path_d_data1,'weight'),3)
ZeroDivisionError: division by zero
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "find_shortest_paths.py", line 432, in <module>
    find_shortest_paths(args.rdfpath,args.modelpath,args.graphpath,graph_types[args.graphtype],args.embedpath,"tfcg",args.fcgtype,args.fcgclass,args.cpu)
  File "find_shortest_paths.py", line 404, in find_shortest_paths
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "find_shortest_paths.py", line 404, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
ZeroDivisionError: division by zero

real	2m17.948s
user	93m43.952s
sys	0m57.648s
